### PR TITLE
merge to 1.16 onPrem installation docs update

### PR DESCRIPTION
**Pull request by lilachmaliniak**
liveController & liveRecorder installation docs

### CODE REVIEW

File: LiveController_deployment.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+## Machine prerequisites:
+====================================================
+liveController & liveRecorder can be installed in single or separate machines.
+
+## Software prerequisites:
+====================================================
+ - RHEL/CentOS 7 or above or Ubuntu 16.0.4 or above
```
Comment (kalturaguy) [inline LiveController_deployment.md]: CentOS 6.4 and above . (this is what we have on Production/QA)

File: LiveController_deployment.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+ ====================================================
+  - deployment/upgradeLive can be used for upgrade
+  > [installation base path]/latest/deployment/upgradeLive vx.y.z
+  alternatively download and install latest version according to following instructions:
+   wget https://github.com/kaltura/liveDVR/archive/v[x.y.z].tar.gz
+   tar -xvzf v[x.y.z].tar.gz
+   mv liveDVR-[x.y.z] [installation base path]
+   rm -f v[x.y.z].tar.gz
+   - install Node Version Manager 6.3.0+
+   - install Node Packaged Modules (npm) 1.4.3+
```
Comment (kalturaguy) [inline LiveController_deployment.md]: don't think we need npm (it's installed with node)

File: liveRecorder/liveRecorder_deployment.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+## Machine prerequisites:
+====================================================
+liveController & liveRecorder can be installed in single or separate machines.
+
+## Software prerequisites:
+====================================================
+ - RHEL/CentOS 7 or above or Ubuntu 16.0.4 or above
```
Comment (kalturaguy) [inline liveRecorder/liveRecorder_deployment.md]: 6.4

File: LiveController_deployment.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+    general configuration:
+    =======================================
+    @LIVE_CONTENT_PATH@ - root path to all live streams content
+ 	@LIVE_ARCHIVE_CONTENT_PATH@ - root path for archived live streams content
+ 	@LOG_FILE_NAME@ - liveController's log filename, including full path
+
+ 	BE server configuration section:
+ 	===================================
+ 	@KALTURA_SERVICE_URL@ - url to access the BE server (example:  "http://my_backend_server.com")
```
Comment (yossipapi) [inline LiveController_deployment.md]: Didn't w talk about using a general @ROOT_DIRECTORY@ path to decrease the amount of required tokens

File: LiveController_deployment.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+ mv liveDVR-[x.y.z] [installation base path]
+ rm -f v[x.y.z].tar.gz
+ - install Node Version Manager 6.3.0+
+ - install Node Packaged Modules (npm) 1.4.3+
+ - create configMapping.json from common/config/configMapping.json.template:
+
+    general configuration:
+    =======================================
+    @LIVE_CONTENT_PATH@ - root path to all live streams content
+ 	@LIVE_ARCHIVE_CONTENT_PATH@ - root path for archived live streams content
```
Comment (yossipapi) [inline LiveController_deployment.md]: Didn't w talk about using a general @ROOT_DIRECTORY@ path to decrease the amount of required tokens

Same applies to @RECORDING_FOLDER@, etc...

**Pull request merged**